lblod/app-subsidiepunt
# Feature: add e-inclusion 'verantwoording' form

## ID
DGS-356

## Description
- Adds the verantwoording form to e-inclusion subsidy
- Updates the enddate and series title

## Type of change


## How to setup
Run the frontend PR https://github.com/lblod/frontend-subsidiepunt/pull/24

## How to test
Run subsidiepunt, create e-inclusion subsidy, fill out both steps and verify if working.
